Cargo.toml

```diff
@@ -30,17 +30,18 @@ required-features = ["tfidf", "textrank"]
 regex = "1.0"
 lazy_static = "1.0"
 phf = "0.11"
-hashbrown = { version = "0.12", default-features = false, features = ["inline-more"] }
```

Comment:
This would need a benchmark result to justify it.

Reply:
I'm wondering if instead HMMModel should be a seprate struct and that Jieba's interface should be expanded to be

```
cut()
hmm_cut()
```

Will discussion back in #48